auth/src/lib.rs

```diff
@@ -66,6 +87,12 @@ pub mod registration {
         pub server_data: String,
         pub registration_upload: opaque::server::registration::RegistrationUpload,
     }
+
+    #[derive(Serialize, Deserialize, Clone)]
+    pub struct ClientConventionalRegistrationRequest {
```

Comment:
I'd like to split this PR into login (that I would more easily approve) and registration/reset, that I would like to keep OPAQUE only if possible. For me, the easy login method is not meant to be opened up to the world, but more like running scripts on the same host more conveniently.

Reply:
So you would basically require any service that is automatically registering users to use OPAQUE?

Reply:
Let's move this conversation to the next PR, since I'd like to anyway split it into one for login and one for registration.